packages/react/src/Page/Page.tsx

```diff
@@ -8,14 +8,25 @@ import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
 import { clsx } from 'clsx'
 import { forwardRef } from 'react'
 
-export type PageProps = PropsWithChildren<HTMLAttributes<HTMLDivElement>>
+export type PageProps = {
+  /*
+   * Whether the page contains a Menu component.
+   * This requires the following class names on the appropriate children:
+   *   - `ams-page__area--skip-link`
+   *   - `ams-page__area--header`
+   *   - `ams-page__area--menu`
+   *   - `ams-page__area--body`
+   *   - `ams-page__area--footer`
+   */
+  hasMenu?: boolean
```

Comment:
Maybe `withMenu` would be even better?

To me, ‘has’ sounds more like the presence could be dynamic. ‘Page with menu’ indicates that there is a fixed menu on the page that doesn’t go anywhere. And it reads slighty better.